design-library/src/components/BccPagination/BccPagination.vue

```diff
@@ -0,0 +1,143 @@
+<script setup lang="ts">
+import { ArrowBackIosIcon } from "@bcc-code/icons-vue";
+import { ArrowForwardIosIcon } from "@bcc-code/icons-vue";
+import { toRefs, ref, computed, watchEffect } from "vue";
+import type { Item } from "../BccTable/BccTable.vue";
+import BccSelect from "../BccSelect/BccSelect.vue";
+import BccFormLabel from "../BccFormLabel/BccFormLabel.vue";
+
+type Props = {
+  tableItems: Item[];
+  rowsPerPageArray?: number[];
```

Comment:
Two small suggestions:
- This prop is optional but it doesn't have a default. I suggest to add one, like 5/10/25/50 or something.
- The name feels a little unclear to me, suggestion: `rowsPerPageOptions`.